plugins/codemirror/deps.json

```diff
@@ -0,0 +1,10 @@
+{
+    "codemirror": [
+        "lib/codemirror.css",
+        "lib/codemirror.js",
+        "mode/xml/xml.js"
+    ],
+    "js-beautify": [
+        "js/lib/beautify-html.js"
+    ]
```

Comment:
Based on https://github.com/liferay/liferay-ckeditor/pull/121/files#r476273541, if we wanted to go with that... we'd probably want to store `js` and `css` deps in different places so we could load them in different ways in an automatic way that doesn't rely on us in coding those paths twice.

Reply:
<a href="https://gitme.me/image?url=https%3A%2F%2Fmedia3.giphy.com%2Fmedia%2FwzD3nQPA4gqHK%2Fgiphy.gif&token=done" data-gitmeme-token="done"><img src="https://media3.giphy.com/media/wzD3nQPA4gqHK/giphy.gif" title="Created by gitme.me with /done"/></a>